### DISCUSSION TITLE
AMD Support?

### DISCUSSION THREAD

**Discussion by smpeyres** [idaholab/moose#25018] [Q&A Getting Started]
I am on the market for a new laptop, debating between AMD- and Intel-based CPU w/ integrated graphics on a Linux OS. Is MOOSE supported on AMD hardware? 

I ask because back when support for Apple's M1 was still in development, someone (don't recall who) mentioned that some dependencies or MOOSE core functionalities were Intel-specific (please correct me if I am wrong!), but have been extended to support Apple silicon now. It also seems INL's recent supercomputing resources have all been Intel, too, which may or may not be related.

Thank you!

**Comment by GiudGiud** [comment]
@milljm @loganharbour will be most informed here

MOOSE works very well on M1 & M2

**Reply by milljm** [reply]
I recall issues in the past with Apple Silicon, but no longer. It's the platform most of the internal team is using these days (to echo what @GiudGiud said).

As for Intel vs AMD for the Linux arches, no functionality reduction that I know of.
